third_party/android_deps/BUILD.gn

```diff
@@ -54,17 +53,6 @@ android_aar_prebuilt("com_github_bumptech_glide_gifdecoder_java") {
   deps = [ "//third_party/androidx:androidx_annotation_annotation_java" ]
 }
 
-android_aar_prebuilt("com_wireguard_android_java") {
-  aar_path = "libs/com_wireguard_android/tunnel-1.0.20250531.aar"
```

Comment:
We should remove the file itself at `third_party/android_deps/libs/com_wireguard_android/tunnel-1.0.20250531.aar`

We can also delete the prior tools script `tools/android/wireguard-build.sh`

Reply:
Fixed in https://github.com/brave/brave-core/pull/31864/commits/26d2ea2c05edf76d1b25435163c80c7473c2dc60